x-pack/plugins/timelines/server/search_strategy/timeline/factory/events/all/index.ts

```diff
@@ -62,7 +62,7 @@ export const timelineEventsAll: TimelineFactory<TimelineEventsQueries.all> = {
       )
     );
 
-    const consumers = buckets.reduce(
+    const consumers: Record<string, number> = producerBuckets.reduce(
```

Comment:
should this be `const producers` then?

Reply:
I did not want to change it because from the producer from rule become the consumer from alerts